automation_tools/__init__.py

```diff
@@ -1467,3 +1471,249 @@ def setenforce(mode):
         ))
 
     run('setenforce {0}'.format(mode))
+
+
+def get_hostname_from_ip(ip):
+    """Retrives the hostname by logging into remote machine by IP.
+    Specially for the systems who doesnt support reverse DNS.
+    e.g usersys machines.
+
+    :param ip: A string. The IP address of the remote host.
+
+    """
+    gethost = lambda: run('hostname')
+    result = execute(gethost, host=ip)
+    return result[ip]
+
+
+def get_OpenstackClient():
+    """Creates client object instance from openstack novaclient API.
+    And returns the client object for further use.
+
+    The following environment variables affect this command:
+
+    USERNAME:
+        The username of an openstack project to login.
+    PASSWORD:
+        The password of an openstack project to login.
+    AUTH_URL:
+        The authentication url of the project.
+    PROJECT_ID:
```

Comment:
Trailing colons are unnecessary.

    USERNAME
        The username of an openstack project to login.
    PASSWORD
        The password of an openstack project to login.
    AUTH_URL
        The authentication url of the project.
    PROJECT_ID